c/ldv-linux-3.14-races/linux-3.14--drivers--media--platform--marvell-ccic--cafe_ccic.ko_false-unreach-call.cil.c

```diff
@@ -0,0 +1,12893 @@
+   struct timespec blkio_start ;
+   struct timespec blkio_end ;
+   u64 blkio_delay ;
+   u64 swapin_delay ;
+   u32 blkio_count ;
+   u32 swapin_count ;
+   struct timespec freepages_start ;
+   struct timespec freepages_end ;
+   u64 freepages_delay ;
+   u32 freepages_count ;
+};
+struct io_context;
+struct pipe_inode_info;
+struct load_weight {
+   unsigned long weight ;
+   u32 inv_weight ;
+};
+struct sched_avg {
+   u32 runnable_avg_sum ;
+   u32 runnable_avg_period ;
+   u64 last_runnable_update ;
+   s64 decay_count ;
+   unsigned long load_avg_contrib ;
+};
+struct sched_statistics {
+   u64 wait_start ;
+   u64 wait_max ;
+   u64 wait_count ;
+   u64 wait_sum ;
+   u64 iowait_count ;
+   u64 iowait_sum ;
+   u64 sleep_start ;
+   u64 sleep_max ;
+   s64 sum_sleep_runtime ;
+   u64 block_start ;
+   u64 block_max ;
+   u64 exec_max ;
+   u64 slice_max ;
+   u64 nr_migrations_cold ;
+   u64 nr_failed_migrations_affine ;
+   u64 nr_failed_migrations_running ;
+   u64 nr_failed_migrations_hot ;
+   u64 nr_forced_migrations ;
+   u64 nr_wakeups ;
+   u64 nr_wakeups_sync ;
+   u64 nr_wakeups_migrate ;
+   u64 nr_wakeups_local ;
+   u64 nr_wakeups_remote ;
+   u64 nr_wakeups_affine ;
+   u64 nr_wakeups_affine_attempts ;
+   u64 nr_wakeups_passive ;
+   u64 nr_wakeups_idle ;
+};
+struct sched_entity {
+   struct load_weight load ;
+   struct rb_node run_node ;
+   struct list_head group_node ;
+   unsigned int on_rq ;
+   u64 exec_start ;
+   u64 sum_exec_runtime ;
+   u64 vruntime ;
+   u64 prev_sum_exec_runtime ;
+   u64 nr_migrations ;
+   struct sched_statistics statistics ;
+   struct sched_entity *parent ;
+   struct cfs_rq *cfs_rq ;
+   struct cfs_rq *my_q ;
+   struct sched_avg avg ;
+};
+struct rt_rq;
+struct sched_rt_entity {
+   struct list_head run_list ;
+   unsigned long timeout ;
+   unsigned long watchdog_stamp ;
+   unsigned int time_slice ;
+   struct sched_rt_entity *back ;
+   struct sched_rt_entity *parent ;
+   struct rt_rq *rt_rq ;
+   struct rt_rq *my_q ;
+};
+struct sched_dl_entity {
+   struct rb_node rb_node ;
+   u64 dl_runtime ;
+   u64 dl_deadline ;
+   u64 dl_period ;
+   u64 dl_bw ;
+   s64 runtime ;
+   u64 deadline ;
+   unsigned int flags ;
+   int dl_throttled ;
+   int dl_new ;
+   int dl_boosted ;
+   struct hrtimer dl_timer ;
+};
+struct memcg_batch_info {
+   int do_batch ;
+   struct mem_cgroup *memcg ;
+   unsigned long nr_pages ;
+   unsigned long memsw_nr_pages ;
+};
+struct memcg_oom_info {
+   struct mem_cgroup *memcg ;
+   gfp_t gfp_mask ;
+   int order ;
+   unsigned int may_oom : 1 ;
+};
+struct sched_class;
+struct files_struct;
+struct css_set;
+struct compat_robust_list_head;
+struct numa_group;
+struct ftrace_ret_stack;
+struct task_struct {
+   long volatile   state ;
+   void *stack ;
+   atomic_t usage ;
+   unsigned int flags ;
+   unsigned int ptrace ;
+   struct llist_node wake_entry ;
+   int on_cpu ;
+   struct task_struct *last_wakee ;
+   unsigned long wakee_flips ;
+ 
```

Comment:
Does this precisely match the definition on the platform the benchmarks will be running on? This is as much a call to @dbeyer as it is to the benchmark submitter; if it isn't, we'll be in big trouble as the benchmark (of course) does not provide definitions of various `pthread_*` functions, which, however, are expected to take arguments of a suitable type.